nucliadb_node/src/writer/grpc_driver.rs

```diff
@@ -521,19 +521,19 @@ impl NodeWriter for NodeWriterGRPCDriver {
 mod tests {
     use std::net::SocketAddr;
 
-    use nucliadb_protos::node_writer_client::NodeWriterClient;
-    use nucliadb_protos::node_writer_server::NodeWriterServer;
+    use nucliadb_core::protos::node_writer_client::NodeWriterClient;
+    use nucliadb_core::protos::node_writer_server::NodeWriterServer;
```

Comment:
Could these `node_writer_client` and `node_writer_server` respectively rename to `client` and `server` ?

Reply:
As before, this is a protos defined module. The suggestion is ok, but it would require changing the protos and therefore growing the PR a lot. Maybe for a separate PR?

Reply:
Ok, fair enough :cat: